SpeechResponder/Properties/SpeechResponder.resx

```diff
@@ -230,4 +230,8 @@
   <data name="reset_script_message" xml:space="preserve">
     <value>Are you sure you want to reset the "{0}" script?</value>
   </data>
+  <data name="name_taken" xml:space="preserve">
+    <value>That name is taken. Please choose another name.</value>
+    <comment>Presented in a message box when the name supplied by the user is already taken by another item.</comment>
```

Comment:
...when the name of a **copied personality** is...?

Reply:
On reflection, I think it would be UX to validate the new name while the dialog is still open. I'll look into doing that.

Reply:
This is probably the best start point <https://docs.microsoft.com/en-us/dotnet/framework/wpf/data/how-to-implement-binding-validation>.

I think I shall close and re-work it.